packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.0.1 - Jun 13, 2022
```

Comment:
According the current test progress, we may delay the release to Jun 15

Reply:
change to 16th, thanks